frontend_tests/node_tests/upload.js

```diff
@@ -492,7 +511,7 @@ test("uppy_events", ({override, override_rewire}) => {
         };
     };
     upload.setup_upload({mode: "compose"});
-    assert.equal(Object.keys(callbacks).length, 5);
+    assert.equal(Object.keys(callbacks).length, 6);
```

Comment:
The new one is `uppy.on("progress"` which we're using instead of having the progressbar handle it.